test/testenv/verificationutils.go

```diff
@@ -375,3 +375,27 @@ func VerifyIndexerClusterScalingUp(deployment *Deployment, testenvInstance *Test
 		return idxc.Status.Phase
 	}, deployment.GetTimeout(), PollInterval).Should(gomega.Equal(splcommon.PhaseScalingUp))
 }
+
+// VerifySecretObjectUpdate Check whether the secret object info is pushed to other pods
```

Comment:
Nit: VerifySecretObjectUpdate checks whether the secret object has an updated value for a key

Reply:
modified

Reply:
@tpereirasplunk you might need to redo this.